Source/MQTTnet/Client/Connecting/MqttClientAuthenticateResult.cs

```diff
@@ -7,42 +7,110 @@ namespace MQTTnet.Client.Connecting
     // TODO: Consider renaming this to _MqttClientConnectResult_
     public class MqttClientAuthenticateResult
     {
+        /// <summary>
+        /// Gets or sets the result code.
+        /// Hint: MQTT 5 feature only.
```

Comment:
This is a good comment because it has an information (MQTT v5 only) which is not visible to the user.